### PR TITLE
Added Enums For Chains

### PR DESCRIPTION
**Pull request by rayyan224**
---
Added Enums for Supported chains.
---

### Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/SECURITY.md).
- [x] My code is conform the [code style](https://github.com/MoralisWeb3/Moralis-JS-SDK/blob/main/CODE_STYLE.md)
- [x] I have made corresponding changes to the documentation
- [x] I have updated Typescript definitions when needed

### Issue Description

Related issue: #`FILL_THIS_OUT`

### CODE REVIEW

**Comment by ivan-liljeqvist** [comment]
@rayyan224 let's put these chains in a separate file
maybe `Chains.js`

**Comment by rayyan224** [comment]
> @rayyan224 let's put these chains in a separate file maybe `Chains.js`

Sure

> @rayyan224 let's put these chains in a separate file maybe `Chains.js`

Ok, np.

File: types/index.d.ts [hunk 1]
```diff
@@ -1808,6 +1824,22 @@ export namespace Moralis {
...
+    ETH_LOCALDEVCHAIN: EChains.ETH_LOCALDEVCHAIN;
+    POLYGON_MAINNET: EChains.POLYGON_MAINNET;
+    POLYGON_MUMBAI: EChains.POLYGON_MUMBAI;
+    BSC_MAINNET: EChains.BSC_MAINNET;
+    BSC_TESTNET: EChains.BSC_TESTNET;
+    AVAX_MAINNET: EChains.AVAX_MAINNET;
+    AVAX_TESTNET: EChains.AVAX_TESTNET;
+    FANTOM_MAINNET: EChains.FANTOM_MAINNET;
+  }
+
```
Comment (ErnoW) [inline types/index.d.ts:1842]:
```suggestion

  let Chains: IChains;
```

Add this to export the Moralis.Chains type correctly

**Review by ErnoW — CHANGES_REQUESTED** [review]

**Review by ErnoW — APPROVED** [review]

**Pull request merged**